src/components/distributions/editor/index.js

```diff
@@ -65,6 +65,24 @@ export default class Guesstimate extends Component{
 
   _changeMetricClickMode(newMode) { this.props.dispatch(changeMetricClickMode(newMode)) }
 
+  handleReturn(shifted) {
+    if (shifted) {
+      this.props.jumpSection()
+    } else {
+      this.props.onReturn()
+    }
+    return true
+  }
+
+  handleTab(shifted) {
+    if (shifted) {
+      this.props.jumpSection()
```

Comment:
Yes, they do in this case, but it felt clearer to me to have them separated based on whether it was a tab or a return rather than whether it was shifted or not.